src/platforms/common/enriching-events/identify-user.mdx

```diff
@@ -17,14 +17,28 @@ Users consist of a few critical pieces of information that construct a unique id
 
 ### `id`
 
+<Note>
+
+If you don't provide an `id`, the SDK falls back to `installationId`, which the SDK randomly generates once during an app's installation.
+
+</Note>
+
 Your internal identifier for the user.
+
 </PlatformSection>
 
 <PlatformSection supported={["apple"]}>
 
 ### `userId`
 
+<Note>
+
+Since version 6.0.1, if you don't provide a `userId`, the SDK falls back to `installationId`, which the SDK randomly generates once during an app's installation.
+
+</Note>
```

Comment:
This only applies to Apple. I don't think it should stay here.

Reply:
It's in a platform section that's only supported for Apple (see line 30), so it will only display for Apple.